libraries/Microsoft.Bot.Builder.Core/IBotContext.cs

```diff
@@ -28,6 +28,8 @@ public interface IBotContext
 
         Task SendActivity(params string[] textRepliesToSend);
         Task SendActivity(params Activity[] activities);
+        Task SendActivity(params IMessageActivity[] activities);
```

Comment:
We shouldn't be simply adding overloads for every possible type.

Reply:
Good points Look for the checkin shortly.